scripts/r.in.wms/wms_cap_parsers.py

```diff
@@ -131,7 +131,7 @@ def _checkFormats(self, capability):
         """!Check if format element is defined."""
         request = self._find(capability, "Request")
         get_map = self._find(request, "GetMap")
-        formats = self._findall(get_map, "Format")
+        self._findall(get_map, "Format")
```

Comment:
Can you please check how the check here is supposed to work? Why these functions are called and how a failed check behaves?

Reply:
What parameter should be used for the testing you requested? I understand that you want the test to fail, but can you suggest a quick way to test or should I write unit tests?

Anyways, I tried testing outside of the 'grass' project since I don't know how to debug and test interactively (GUI) since it requires a specifically formatted XML file. At least I thought so...

Here is how I tested, but you may be able to suggest a much easier method. I think the easiest way would be to use a raster WebMap service URL that doesn't have "Format" tags. GUI can be used to test it. 

step 1: Get XML and save it as XML file.
```r.in.wms -c url="http://ows.mundialis.de/services/service?" > "/home/mshukun/Downloads/wms.xml" 2>&1```

Step 2: Delete the "Format" tags under "GetMap" tag from wms.xml
```xml
<GetMap>
  <Format>image/jpeg</Format>  # Delete 
  <Format>image/png</Format>   # Delete
  <DCPType>
  <HTTP>
    <Get>
      <OnlineResource xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://ows.mundialis.de/osm/service?"/>
    </Get>
  </HTTP>
  </DCPType>
</GetMap>
```
Step 3: Create etreeElement object
```
tree = ET.parse("wms.xml")
root = tree.getroot()
```

Step 4:
In order to make "_find", "_findall", and "checkFormats" callable outside of the `grass`, I copied and pasted them outside of the project. 

```checkFormats(root.find("Capability"))```

Result:
```
    raise ParseError(

xml.etree.ElementTree.ParseError: Unable to parse capabilities file. 

                            Tag 'Format' was not found.
```

Reply:
No need to test it, it's clear from the code. `_checkFormats` will raise ParseError, so technically, the current solution is correct, although the docstring should mention that the method throws exception when it won't find formats.

Reply:
> No need to test it, it's clear from the code. `_checkFormats` will raise ParseError, so technically, the current solution is correct, although the docstring should mention that the method throws exception when it won't find formats.

This was rather simple fix so I might misinterplet @wenzeslaus question.